src/clr-angular/accordion/accordion-panel.html

```diff
@@ -0,0 +1,48 @@
+<ng-container *ngIf="panel | async; let panel">
+  <div *ngIf="panel.status !== AccordionStatus.Inactive" aria-live="assertive" class="clr-screen-reader-only">
+    <ng-container *ngIf="panel.status === AccordionStatus.Error">{{commonStrings.danger}}</ng-container>
+    <ng-container *ngIf="panel.status === AccordionStatus.Complete">{{commonStrings.success}}</ng-container>
+  </div>
+
+  <div role="group" [ngClass]="getPanelStateClasses(panel)">
+    <div class="clr-accordion-header" [class.focus-within]="focusHeader">
+      <button
+        type="button"
+        class="clr-accordion-header-button"
+        (click)="togglePanel()"
+        (focus)="focusHeader = true"
+        (blur)="focusHeader = false"
+        [id]="'clr-accordion-header-' + panel.id + panel.accordionId"
+        [attr.aria-controls]="'clr-accordion-content-' + panel.id + panel.accordionId"
+        [attr.aria-expanded]="panel.open"
+        [disabled]="panel.disabled"
```

Comment:
Could users disable it declaratively in the template?

Reply:
Added new Input as well as unit tests